packages/core/src/shared/i18n/package.nls.json

```diff
@@ -265,5 +265,51 @@
     "AWS.amazonq.learnMore": "Learn More About Amazon Q",
```

Comment:
Why is this file being moved? This file should remain at the root beside package.json

Reply:
I see. I just changed because what I've saw from the previous conversation the suggestion was to include it in a shared folder? Anyway, I can add it back.

Reply:
Ah my bad, I just meant you could look into using your i18n utils into the shared folder and then keep the package.nls.json in the same place

Reply:
Oh I see 😅 This makes sense. I'll include over there.